src/pages/Inventory.page.js

```diff
@@ -9,7 +9,32 @@ export class InventoryPage extends BaseSwagLabPage {
 
     addItemToCartButton = this.page.locator('[id^="add-to-cart"]');
 
+    sortProductButton = this.page.getByTestId('product-sort-container');
+
+    inventoryItemsPrice = this.page.getByTestId('inventory-item-price');
+
     async addItemToCartById(id) {
         await this.addItemToCartButton.nth(id).click();
     }
+
+    async sortProductNameAZ() {
```

Comment:
Cool.
Now let's combine all this 4 functions into 1 function with parameter - sortOption.
You will have something like:
`performProductSort(sortOption)`
then in test you will be able to use it like: `performProductSort("az")` / `performProductSort("hilo")` / etc